src/verifier/builder_visitor.rs

```diff
@@ -73,7 +73,7 @@ impl SymbolTableBuilder<'_> {
 
     fn add_provider(&mut self, provider: &Provider) {
         if self.table.lookup(&provider.name).is_some() {
-            // This should never be the case since it's controlled by the compiler!
+            // This should never be the case since it's controlled by the compiler! - THIS BREAKS WITH WHAMM
```

Comment:
What's this added comment for?

Reply:
so we know that this is the point of failure for probes with whamm as their provider. If its not useful, I can remove it